WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPlaceholder.swift

```diff
@@ -1,5 +1,8 @@
 // Empty state for Stock Photos
 final class StockPhotosPlaceholder: WPNoResultsView {
```

Comment:
Another comment: I'd prefer to see this use [`NoResultsViewController`](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift) since it's swift, whereas `WPNoResultsView` is ObjC.

Reply:
I am with Eduardo on this one. The picker requires a view, if the reason to suggest using `NoResultsViewController` maybe we should migrate `WPNoResultsView` to Swift, and make `NoResultsViewController` use it, in a separate PR?

Reply:
Another thing to consider: I think we want the touch target to be at least 44px tall but currently it's only as tall as the text. I'm not sure how you'd prefer to fix that, but I wonder if a `UIButton` (possibly a `SubheadlineButton`?) would work best. And in that case, you'd probably want logic to use either a `UIButton` or a `UILabel`. And once you have logic like that, you want a `UIViewController` and not a `UIView` 😉

I think view controller composition is a better fit here. Don't be afraid of it!

Reply:
I completely agree with @nheagy . I did try using the `NoResultsViewController` at the beginning :)

But for support a controller as the MediaPicker empty view will take some extra work. I'd prefer to continue using `WPNoResultsView` in this way for now until we make the effort to deprecate it in favor of `NoResultsViewController`.

I'll be happy to make the necessary changes in the MediaPicker project to make this happen!